nevergrad/optimization/optimizerlib.py

```diff
@@ -1755,3 +1756,70 @@ def _internal_ask_candidate(self) -> p.Parameter:
 
     def _internal_tell_candidate(self, candidate: p.Parameter, value: float) -> None:
         self._optim.tell(candidate, value)
+
+
+@registry.register
+class Ctulo(base.Optimizer):
```

Comment:
why don't you rename it directly NGO2?

Reply:
Ok I do so.

Reply:
Well NGO10 because we have had several variants with various numbers.